pom.xml

```diff
@@ -19,6 +19,11 @@
         <!--examples-->
         <module>examples/processors</module>
         <module>docker</module>
+        <!-- test jar builders -->
+        <module>streams/service/src/test/resources/custom-udf-microtest</module>
```

Comment:
Okay, that seems a good place to put them, I'll try that.  If I can't prevent it from including all the super-module dependencies, I'll have to put it at the top level.